addons/resource.language.en_gb/resources/strings.po

```diff
@@ -900,11 +900,13 @@ msgctxt "#203"
 msgid "Plot outline"
 msgstr ""
 
+#. label of a smartplaylist filter option to filter for albums on compilations
 #: xbmc/playlists/SmartPlaylist.cpp
 msgctxt "#204"
 msgid "Compilation"
 msgstr ""
 
+#. label of a smartplaylist filter option to filter for songs or albums on votes for rating
```

Comment:
ah ok, so really votes from the metadata sites. Just wasn't sure as I was not aware of any voting capability or info in Kodi, but haven't used music section lately. Sorry for the noise.